src/style/stylize.rs

```diff
@@ -23,6 +25,75 @@ pub trait Styled {
     fn set_style<S: Into<Style>>(self, style: S) -> Self::Item;
 }
 
+/// A helper struct to make it easy to debug using the `Stylize` method names
+pub(crate) struct StylizeDebug {
```

Comment:
name => ColorDebug perhaps and ColorDebugKind below.

Reply:
a75abe5ef1b24633cab841336547ac7df797ca02